src/frontend/src/flows/recovery/confirmSeedPhrase.ts

```diff
@@ -156,6 +169,7 @@ export const wordTemplate = ({
       type="text"
       class="c-recoveryInput"
       ${ref(word.elem)}
+      ?autofocus=${word.autofocus === true}
```

Comment:
dumb question: what's the question mark `?`? is that a lit thing?

Reply:
[yes!](https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions)

Reply:
@peterpeterparker yes it sets the property only if the computed value is true(isch)